masterlist.yaml

```diff
@@ -1843,21 +1853,130 @@ plugins:
         name: 'Immersive Citizens - AI Overhaul SE on Nexus Mods'
       - link: 'https://bethesda.net/en/mods/skyrim/mod-detail/3016198'
         name: 'Immersive Citizens - AI Overhaul [PC] on Bethesda.net'
-    group: *alternateStartGroup
```

Comment:
@Arthmoor Requested that this should be looked at and moved higher in the sorting. I've done a lot of testing with this over the last few days and I have to agree, it conflicts with any mods that alters locations.